core/domain/acl_decorators.py

```diff
@@ -989,6 +989,26 @@ def test_can_create_topic(self, **kwargs):
     return test_can_create_topic
 
 
+def can_access_topics_and_skills_dashboard(handler):
+    """Decorator to check whether the user can create a topic."""
+
+    def test_can_access_topics_and_skills_dashboard(self, **kwargs):
+        if not self.user_id:
+            raise self.NotLoggedInException
+
+        user_actions_info = user_services.UserActionsInfo(self.user_id)
+
+        if role_services.ACTION_EDIT_ANY_TOPIC in user_actions_info.actions:
```

Comment:
Please double-check this with @tjiang11's doc (with regards to who can see this dashboard).

Reply:
Done, added access for all topic managers and admins. Created a new action for the same.